common/lib/xmodule/xmodule/library_content_module.py

```diff
@@ -146,6 +171,13 @@ class LibraryContentFields(object):
         default=1,
         scope=Scope.settings,
     )
+    capa_type = String(
+        display_name=_("Problem Type"),
```

Comment:
To be clear, there may be > 1 item retrieved from the content library. But if those items include problems, all problems must be of the specified type?